src/make_round_config.R

```diff
@@ -1 +1,162 @@
-print("Once implemented, script will update hub-config/tasks.config based on latest list of clades to model.")
\ No newline at end of file
+## Add a new round to the hub's task config (hub-config/tasks.json)
+## This script is part of a "round open" process run via a scheduled GitHub action and
+## should be run from the hub's src/ directory.
+
+library(hubAdmin)
+library(hubUtils)
+library(tools)
+
+get_clade_file_info <- function(hub_dir) {
+  # the round_id is the date of the most recent "clades to model"
```

Comment:
I don't see a round_id referenced in this file.

Reply:
I updated the documentation to make it clearer that `round_id` isn't in the context of the clades list file. Rather, we're using the clade list filename (e.g., `2024-09-25.txt`) to derive the id for the new round being created.